README.md

```diff
@@ -93,6 +93,22 @@ client
   });
 ```
 
+### Getting permissions list
+
+```javascript
+const contentSourceKey = ""; // your content source key
+const pageParams = { current: 2, size: 20 }; // current page and page size (optional argument)
+
+client
+  .getPermissions(contentSourceKey, pageParams)
```

Comment:
I've added "operationId" to the Open API specs. This is a good reference for what method names should be named in the generated clients. Let's use these for consistency across the clients: https://github.com/swiftype/website/blob/jasonstoltz-add-operationId-to-swagger/swagger/v1/frito_pie.json#L367

This is good, I just wanted to point it out 👍

Reply:
Yeah, that's a good idea 👍